regparser/tree/xml_parser/reg_text.py

```diff
@@ -177,9 +178,21 @@ def build_from_section(reg_part, section_xml):
                 n = Node(node_text[0], [], [m], source_xml=ch)
                 n.tagged_text = unicode(node_text[1])
                 nodes.append(n)
+
             if node_text[0].endswith('* * *'):
                 nodes.append(Node(label=[mtypes.INLINE_STARS]))
 
+    # special handling for reg J, whose definitions are inexplicably
+    # unnumbered and yet required to belong to node that precedes them
+    if reg_part == '1010' and nodes != [] and nodes[-1].text.find('As used in this part:') > -1:
+        for i, [plain_text, tagged_text] in enumerate(section_texts):
+            subnode = Node(plain_text, label=['b', str(i)], title="")
+            subnode.tagged_text = tagged_text
+            nodes[-1].children.append(subnode)
```

Comment:
I think you can get to the same outcome by special casing the regulation's _form_ without one-offing reg J. Here, it looks like the form you're trying to match is "we're in the reg text, there is no paragraph marker, there is a definition". Turns out that rule is common across lots of regs, so the "special" case would be generally useful.